### PR TITLE
Doc Fixit: mention how to get protoc compiler in base INSTALL.md

### LINKED ISSUE #5243: Doc Fixit: Update grpc/INSTALL to drop references to OpenSSL and make correct references to hurried installs
https://github.com/grpc/grpc/blob/release-0_13/INSTALL is out of date.
There are references to apt-get install libgrpc, openssl and zlibg we can remove.
This doc needs to be updated for the 0.13 release.

/cc @ctiller @nicolasnoble

### PR DESCRIPTION
**Pull request by stanley-cheung**
Fixes #5243

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -32,6 +32,16 @@ terminal:
...
 
+##Protoc
+
+By default gRPC uses [protocol buffers](https://github.com/google/protobuf),
+you will need the `protoc` compiler to generate stub server and client code.
+
+If you compile from source, see below, the Makefile will automatically try
+and compile the one present in third_party if you cloned the repository
+recursively, and that it detects your system is lacking it.
+
```
Comment (LisaFC) [inline INSTALL.md]: If you compile gRPC from source, as described below, the Makefile will automatically try
and compile the `protoc` in third_party if you cloned the repository
recursively and it detects that you don't already have it installed.

**Comment by LisaFC** [comment]
Does installing both gRPC and protoc as described in this doc get you the gRPC C++ plugin for protoc?

(Also this doc still seems to suggest that building gRPC from source is optional/only if you want to contribute, while AFAIK this is currently the **only** way to install it for C++?)

File: INSTALL.md (hunk 1 above)
Comment (stanley-cheung) [inline INSTALL.md]: Fixed, thanks for the edit.

**Comment by stanley-cheung** [comment]
@ctiller @nicolasnoble Can you guys comment on Lisa's question above?

My understanding is that

- Following the "compile from source" instructions on the base gRPC install file (i.e. clone grpc, submodule update, make and make install) *is* the way to install the gRPC library *and* protoc for C++, no matter whether you want to contribute or just want to use the library.
- Debian has the libgrpc-dev package, but it hasn't been updated to 0.13 yet and it doesn't serve other platform anyways

Is that correct?

**Comment by stanley-cheung** [comment]
I confirmed that compile from source is the only way to install for C++ for now.

**Comment by stanley-cheung** [comment]
@LisaFC Is this good to go?

**Comment by LisaFC** [comment]
Nearly!

**Comment by LisaFC** [comment]
Actually, I think the information I want should probably go in in a separate PR, so I'll merge this for now.

**Comment by LisaFC** [comment]
Just as a follow-up, did you confirm that building gRPC from source also gets you the C++ code generator for protoc - like, if you follow the instructions here, build gRPC, get protoc, you don't need to do anything else to generate stub code?

**Comment by stanley-cheung** [comment]
sounds good, thanks.

**Comment by stanley-cheung** [comment]
That's correct, I verified that.

**Pull request merged**